lib/improver/cli/estimate_emos_coefficients.py

```diff
@@ -169,6 +246,52 @@ def process(historic_forecast, truth, distribution, cycletime, units=None,
             contains a coefficient_index dimension coordinate and a
             coefficient_name auxiliary coordinate.
     """
+    # Support the initial cycles when using a training dataset, where there
+    # might not yet be any available historic forecasts and truth to calibrate
+    # with. In this instance, only a warning is raised, as this would be
+    # expected behaviour when no training dataset exists.
+    if not any([historic_forecast, truth, combined]):
+        msg = ("In order to calculate the EMOS coefficients then either "
+               "the historic_forecast {} and the truth_filepath {} "
+               "should be specified, or the combined_filepath {} should be "
+               "specified alongside the historic_forecast_identifier {} and "
+               "truth_identifier {}. In this case the arguments provided "
+               "were not sufficient.".format(
+                    historic_forecast, truth, combined,
+                    historic_forecast_dict, truth_dict))
+        warnings.warn(msg)
+        return
+
+    if any([historic_forecast, truth]):
```

Comment:
Can we have comments above each of these blocks of if statements?

Reply:
I've added some comments around these if statements.